lib/stripe/connect/oauth.ex

```diff
@@ -45,22 +45,23 @@ defmodule Stripe.Connect.OAuth do
   @doc """
   Execute the OAuth callback to Stripe using the code supplied in the request parameter of the oauth redirect at the end of the onboarding workflow.
 
-## Example
-```
-{:ok, resp} = Stripe.Connect.token code
-IO.inspect resp
-%Stripe.Connect.OAuth.TokenResponse{
-  access_token: "ACCESS_TOKEN",
-  livemode: false,
-  refresh_token: "REFRESH_TOKEN",
-  scope: "read_write",
-  stripe_publishable_key: "PUBLISHABLE_KEY",
-  stripe_user_id: "USER_ID",
-  token_type: "bearer"
-}
-
-```
+  ## Example
+  ```
+  {:ok, resp} = Stripe.Connect.Oauth.token(code)
+
+  IO.inspect resp
```

Comment:
Unless what you're saying is that the above renders the below? It might be better to indicate with an

```
iex> {:ok, resp} = Stripe.Connect.OAuth.token(code)
...> IO.inspect resp
%Stripe.Connect.OAuth.TokenResponse{...}
```

example